test/distributed/org/apache/cassandra/distributed/test/streaming/BoundExceptionTest.java

```diff
@@ -75,11 +68,9 @@ public void testExceptionCycle()
 
         StringBuilder boundedStackTrace = StreamSession.boundStackTrace(e1, LIMIT);
         String expectedStackTrace = "java.lang.Exception: Test exception 1\n" +
-                                     "\torg.apache.cassandra.distributed.test.streaming.BoundExceptionTest.testExceptionCycle(BoundExceptionTest.java:70)\n" +
+                                     "\torg.apache.cassandra.distributed.test.streaming.BoundExceptionTest.testExceptionCycle(BoundExceptionTest.java:63)\n" +
                                      "\tjava.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n" +
                                      "java.lang.RuntimeException: Test exception 2\n" +
-                                     "\torg.apache.cassandra.distributed.test.streaming.BoundExceptionTest.testExceptionCycle(BoundExceptionTest.java:71)\n" +
-                                     "\tjava.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n" +
                                      "[CIRCULAR REFERENCE: java.lang.Exception: Test exception 1]";
```

Comment:
why is there no newline?  to be consistent you should always have a newline or never; but do not mix